.gitlab-ci.yml

```diff
@@ -455,63 +405,63 @@ test_windows_ltsc2022_x64:
 
 release_deb_x64:
   extends: .release
-  needs: ["test_deb_x64"]
+  needs: ["trigger_tests"]
```

Comment:
I think you can move the `needs` into the `.release` section, which would reduce the duplicated boilerplate